# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../linux26/05-devfs-not-exp.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./fs/Kconfig~	2003-05-05 01:53:32.000000000 +0200
+++ ./fs/Kconfig	2003-06-07 17:34:34.000000000 +0200
@@ -738,8 +738,8 @@
 	  programs depend on this, so everyone should say Y here.
 
 config DEVFS_FS
-	bool "/dev file system support (OBSOLETE)"
-	depends on EXPERIMENTAL
+	bool "/dev file system support (Required for ROCK Linux)"
+	default y
 	help
 	  This is support for devfs, a virtual file system (like /proc) which
 	  provides the file system interface to device drivers, normally found
@@ -758,6 +758,7 @@
 config DEVFS_MOUNT
 	bool "Automatically mount at boot"
 	depends on DEVFS_FS
+	default y
 	help
 	  This option appears if you have CONFIG_DEVFS_FS enabled. Setting
 	  this to 'Y' will make the kernel automatically mount devfs onto /dev
